src/internal/constants.ts

```diff
@@ -6,6 +6,11 @@ export const version = '1.10.0';
  */
 export const defaultSDKVersionForCompatCheck = '1.6.0';
 
+/**
+ * This is the SDK version when selectMedia API - VideoAndImage is supported on mobile.
```

Comment:
could you please sync with @SonalJha I think we are moving to plan where we don't need to use API versioning.

Reply:
Video capability is not planned for web and desktop. For mobiles we are keeping the versioning changes as it is for now. As this is an existing API (SelectMedia) a new constant is needed.